src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,110 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final Integer ZERO = 0;
+    private static final Integer BIT_SHIFT_BY_ONE = 1;
+    private boolean wasAdded;
```

Comment:
Smart move, I'd rather not do the same, but looks interesting.  But give it decent name